common/include/pcl/common/impl/bivariate_polynomial.hpp

```diff
@@ -40,6 +40,12 @@
 #define BIVARIATE_POLYNOMIAL_HPP
 
 #include <algorithm>
+#include <cmath>
+#include <fstream>
+#include <iostream>
+#include <istream>
+#include <ostream>
```

Comment:
```suggestion
#include <iostream>
```

Reply:
```suggestion
#include <fstream>
#include <iostream>
```
[fstream](https://en.cppreference.com/w/cpp/header/fstream) doesn't include [iostream](https://en.cppreference.com/w/cpp/header/iostream) if that's what you mean.